sdks/python/apache_beam/transforms/ptransform_test.py

```diff
@@ -1695,11 +1702,18 @@ def test_mean_per_key_pipeline_checking_violated(self):
        | 'EvenMean' >> combine.Mean.PerKey())
       self.p.run()
 
-    self.assertEqual(
-        "Type hint violation for 'CombinePerKey(MeanCombineFn)': "
-        "requires Tuple[TypeVariable[K], Union[float, int, long]] "
-        "but got Tuple[str, str] for element",
-        e.exception.args[0])
+    if sys.version_info[0] >= 3:
+      expected_msg = \
+        "Type hint violation for 'CombinePerKey(MeanCombineFn)': " \
+        "requires Tuple[TypeVariable[K], Union[float, int]] " \
```

Comment:
Why `long` is missing here in Python 3? A comment may help people understand the branching here.

Reply:
In Python 3, short integers are gone in Python 3 and `long` has become `int`.